### DISCUSSION TITLE
best settings

### DISCUSSION THREAD

**Discussion by mr-qansuh** [staxrip/staxrip#805] [Q&A]
I work on a site that combines movies and subtitles together
I want to use settings to produce all new movies in fast time and high quality from 1080p to 360p

**Comment by jessielw** [comment]
Settings is something as the user you need to figure out. You can make presets for each one. However, messing with entire seasons/subtitles will require you to understand some sort of scripting etc. 

Doing it a movie/episode at a time you can easily manipulate everything you need within staxrip. 

Honestly, if you want fast/quick stick to the presets of x264. If you want high quality over speed then start with the preset veryslow.

**Reply by mr-qansuh** [reply]
how can i know the presets from mediainfo

Video
ID                          : 1
Format                      : AVC
Format/Info                 : Advanced Video Codec
Format profile              : High@L4.1
Format settings             : CABAC / 9 Ref Frames
Format settings, CABAC      : Yes
Format settings, Reference  : 9 frames
Codec ID                    : avc1
Codec ID/Info               : Advanced Video Coding
Duration                    : 24 min 42 s
Bit rate                    : 415 kb/s
Width                       : 1 280 pixels
Height                      : 720 pixels
Display aspect ratio        : 16:9
Frame rate mode             : Constant
Frame rate                  : 25.000 FPS
Color space                 : YUV
Chroma subsampling          : 4:2:0
Bit depth                   : 8 bits
Scan type                   : Progressive
Bits/(Pixel*Frame)          : 0.018
Stream size                 : 73.4 MiB (82%)
Writing library             : x264 core 161 r3053 c347e7a
Encoding settings           : cabac=1 / ref=9 / deblock=1:1:1 / analyse=0x3:0x133 / me=umh / subme=10 / psy=1 / psy_rd=0.40:0.00 / mixed_ref=1 / me_range=24 / chroma_me=1 / trellis=2 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=-2 / threads=12 / lookahead_threads=3 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=10 / b_pyramid=2 / b_adapt=2 / b_bias=0 / direct=3 / weightb=1 / open_gop=0 / weightp=2 / keyint=100 / keyint_min=10 / scenecut=40 / intra_refresh=0 / rc_lookahead=60 / rc=crf / mbtree=1 / crf=22.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / ip_ratio=1.40 / aq=1:0.60
Codec configuration box     : avcC

**Reply by jessielw** [reply]
That's a custom setup. Very slow will produce better results.
